Mobile/starter_project_dartsmiths/lib/features/onboarding/articlePage.dart

```diff
@@ -0,0 +1,77 @@
+
+import 'package:flutter/material.dart';
+
+import '../../core/shared_widgets/colors.dart';
+import 'nextPointer.dart';
+
+class ArticlePage extends StatefulWidget {
+  const ArticlePage({super.key});
+
+  @override
+  State<ArticlePage> createState() => _ArticlePageState();
+}
+
+class _ArticlePageState extends State<ArticlePage> {
+  final List<List<Color>> nexPointercolor = [
+    [nextPointerColor1, nextPointerColor2, nextPointerColor3],
+    [nextPointerColor2, nextPointerColor1, nextPointerColor3]
+  ];
+  // Sizes for the next pointer width and height
+
+  final List<List<List<double>>> sizes = [
+    [
+      [8, 23],
+      [8, 8],
+      [8, 8]
+    ],
+    [
+      [8, 8],
+      [8, 23],
+      [8, 8]
+    ],
+  ];
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(

```

Comment:
Please take look at the Flutter best practices doc section 9 regarding containers.

Reply:
@BekyH can you specify the thing that needs fixing first and then suggest the section please? It will be better and faster to fix.

Reply:
Container widget is hefty weight will have an adverse effect on performance. Therefore, using alternate widgets like sizedBox or decoratedBox is recommended.

Reply:
That makes sense but keep in mind it is a preference for widgets not a must currently.